mithril-aggregator/src/services/epoch_service.rs

```diff
@@ -85,9 +103,9 @@ pub trait EpochService: Sync + Send {
 
 struct EpochData {
     epoch: Epoch,
-    protocol_parameters: ProtocolParameters,
-    next_protocol_parameters: ProtocolParameters,
-    upcoming_protocol_parameters: ProtocolParameters,
+    epoch_settings: AggregatorEpochSettings,
+    next_epoch_settings: AggregatorEpochSettings,
+    upcoming_epoch_settings: AggregatorEpochSettings,
```

Comment:
```suggestion
    signer_registration_epoch_settings: AggregatorEpochSettings,
```

Reply:
Shouldn't we leave this refactor in the [ticket](https://github.com/input-output-hk/mithril/issues/1966) dedicated to it? This avoids cases like:
```
    fn upcoming_protocol_parameters(&self) -> StdResult<&ProtocolParameters> {
        Ok(&self
            .unwrap_data()?
            .signer_registration_epoch_settings
            .protocol_parameters)
    }
```

Reply:
Yes, I added the renaming of the `_epoch_settings` functions in the issue #1966 :+1: